assignments/arrays.js

```diff
@@ -75,30 +324,59 @@ let inventory = [
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*`);
+console.log(`car ${inventory[32].id}.id is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}`);
```

Comment:
hey, this works, but the exercise was using the for loop to do it. so think how would you do it by looking at the examples provided above